src/modules/Hero/Hero.tsx

```diff
@@ -1,14 +1,95 @@
 import React from "react";
 import Section from "../components/Section";
 import Flare from "../components/Flare";
-
+import Image from "next/image";
+import barcampHero from "../../../public/images/barcampHero.png";
+import { useState } from "react";
+import { useEffect } from "react";
+import Link from "next/link";
+import Marquee from "react-fast-marquee";
+import { Inter } from "next/font/google";
+const inter = Inter({ subsets: ["latin"] });
 const Hero = () => {
+  const [countdown, setCountdown] = useState<{
+    days: string;
+    hours: string;
+    minutes: string;
+    seconds: string;
+  }>({
+    days: "0",
+    hours: "0",
+    minutes: "0",
+    seconds: "0",
+  });
+
+  useEffect(() => {
+    const pad = (n: number) => (n < 10 ? `0${n}` : n.toString());
+    const targetDate = new Date("2023-11-25T00:00:00Z"); // 25/11/2023
+
+    const intervalId = setInterval(() => {
+      const now = new Date().getTime();
+      const difference = targetDate.getTime() - now;
+
+      const days = Math.floor(difference / (1000 * 60 * 60 * 24));
+      const hours = Math.floor(
+        (difference % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60),
+      );
+      const minutes = Math.floor((difference % (1000 * 60 * 60)) / (1000 * 60));
+      const seconds = Math.floor((difference % (1000 * 60)) / 1000);
+
+      setCountdown({
+        days: pad(days),
+        hours: pad(hours),
+        minutes: pad(minutes),
+        seconds: pad(seconds),
+      });
+
+      if (difference < 0) {
+        clearInterval(intervalId);
+        setCountdown({ days: "00", hours: "00", minutes: "00", seconds: "00" });
+      }
+    }, 1000);
```

Comment:
on initial page load timer not show because it wait for 1 second to run 

solution 
make this in to function and run first time before put it in setInterval
like this
const fn = () => { ... }
fn()
const intervalId = setInterval(fn,1000)

Reply:
Got it Kub .👍